nemo/collections/nlp/data/language_modeling/megatron/dataset_utils.py

```diff
@@ -210,17 +210,21 @@ def create_masked_lm_predictions(
     mask_id,
     max_predictions_per_seq,
     np_rng,
-    max_ngrams=3,
-    do_whole_word_mask=True,
-    favor_longer_ngram=False,
-    do_permutation=False,
+    max_ngram_size=10,
+    mean_ngram_size=None,
+    whole_word_masking=True,
+    favor_long_ngrams=False,
+    permutation=False,
     geometric_dist=False,
```

Comment:
The old `max_ngrams` defaults is 3 now it becomes `max_ngram_size=10`. It will make bert dataset behavior differently

Reply:
Fixed this and set it back to the old defaults.